ui/src/features/components/WebhooksList/CollapsibleWebhook.js

```diff
@@ -117,6 +117,9 @@ export class CollapsibleWebhook extends React.Component {
 		}
         else if (webhook.format_type === 'teams') {
             icons.push('fa-windows');
+        } 
+        if (webhook.format_type === 'discord') {
+            icons.push('fa-discord');
```

Comment:
When I ran the UI I do not see the discord icon, is it on my side only?

![image](https://user-images.githubusercontent.com/19842756/96928988-10066e80-14c2-11eb-98d7-290d5e7391a5.png)